etcdutils.go

```diff
@@ -152,28 +148,29 @@ func RemoveCerts(etcdStaticResourceDir string) {
 	}
 }
 
-func RestoreSnapshot() {
-	// NOT needed, direct etcdctl call
-}
+func PatchManifest(manifestFilePath, old, new string) {
 
-func PatchManifest() {
-}
+	read, err := ioutil.ReadFile(manifestFilePath)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "%s:read error %s", manifestFilePath, err)
```

Comment:
nit: use %v for printing errors

Reply:
Yeah. I used %v elsewhere, but been inconsistent.